qtlocation fix "error: 'runtime_error' is not a member of 'std'"
Patch by Simon Branford, University of Birmingham
--- qtlocation/src/3rdparty/mapbox-gl-native/platform/default/bidi.cpp.orig	2021-01-14 19:41:10.485471000 +0000
+++ qtlocation/src/3rdparty/mapbox-gl-native/platform/default/bidi.cpp	2021-01-14 19:41:18.997592000 +0000
@@ -5,6 +5,7 @@
 #include <unicode/ushape.h>
 
 #include <memory>
+#include <stdexcept>
 
 namespace mbgl {
 
